login-workflow/docs/language-support.md

```diff
@@ -29,10 +32,11 @@ https://www.yourwebsite.com/page?lgn=fr
 ### 3. Local Storage
 
 You can also manually modify the value that is stored in Local Storage to edit or remove the stored value. Removing the value will tell the application to default to the browser default value.
+All applications using the @pxblue/react-auth-workflow will use the same localstorage key when doing this lookup. In order to avoid possible interference with other applications, we recommend using the ChangeLanguage function (#1) instead of relying on localstorage.
```

Comment:
I would move this to the end of this section (after the other callout) and do it as a callout: "> All App....."